translations/monero_it.ts

```diff
@@ -2745,12 +2745,12 @@ Outputs per *: </source>
     <message>
         <location filename="../src/wallet/wallet_args.cpp" line="172"/>
         <source>Logging to: </source>
-        <translation>Loggando in: </translation>
+        <translation>Sto salvando il Log in: </translation>
```

Comment:
I'm not sure but i thinkoriginal was correct. It should mean "sto eseguendo il login in"